tests/test_parser.py

```diff
@@ -1103,3 +1111,24 @@ def test_parameter_braces(self):
               ])
             ),
         )
+    def test_cases(self):
```

Comment:
does this not pass at the moment?

Reply:
Nope, I had omitted (or added? I can't remember now) a parenthesis in the previous function, but undoing that fixed.